### PR TITLE
feat / sequelize models n relations

### PR DESCRIPTION
**Pull request by rogerknl**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
[the remark this answers is not in the document]
Comment (arol) [reply config/config.json]: Use `dotenv` and add `.env` file into the gitignore list

File: package.json [hunk 2]
```diff
@@ -28,7 +28,10 @@
     "koa-bodyparser": "^4.2.1",
     "koa-logger": "^3.2.0",
     "koa-router": "^7.4.0",
-    "nodemailer": "^4.6.7"
+    "mysql2": "^1.5.3",
+    "nodemailer": "^4.6.7",
+    "sequelize": "^4.38.0",
+    "sequelize-cli": "^4.0.0"
```
Comment (arol) [inline package.json]: What does the cli in the dependencies list? This goes to the devDependencies, but your not even using it, so no.

File: routes.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 'use strict';
 const router = require('koa-router')();
 
+const userCont = require( __dirname + '/controllers/userController' );
+
 
 
 router
-  .get('/', (ctx,next)=>ctx.body = "hello world");
+  .get('/', (ctx,next)=>ctx.body = "hello world")
```
Comment (arol) [inline routes.js]: JavaScript ===== single quotes

File: package.json (hunk 2 above)
Comment (rogerknl) [reply package.json]: i used it for migration...

**Pull request merged**